guides/common/modules/proc_importing-ostree-content.adoc

```diff
@@ -0,0 +1,73 @@
+[id="importing-ostree-content_{context}"]
+= Importing OSTree Content
+
+You can import and synchronize OSTree content from custom online sources over HTTP(S).
+
+.Prerequisite
+* A published HTTP location (upstream URL) for the OSTree to import.
+
+.Procedure
+. In the {Project} web UI, navigate to *Content* > *Products* and click *Create Product*.
```

Comment:
We have an attribute for that: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L87